README.adoc

```diff
@@ -324,7 +324,7 @@ The port number from the configuration is retrieved by the [hotspot=33 file=0]`i
 
 To determine whether the inventory service is in maintenance or not (according to the configuration value), [hotspot=55 file=0]`inventoryConfig.isInMaintenance()` class method is used.
 If you set the [hotspot=4 file=1]`io_openliberty_guides_inventory_inMaintenance` property to `true` in the configuration, the inventory service returns the message, `ERROR: Service is currently in maintenance`, along with the contact email.
-The email configuration value can be get by calling [hotspot=60 file=0]`inventoryConfig.getEmail()` method.
+The email configuration value can be gotten by calling [hotspot=60 file=0]`inventoryConfig.getEmail()` method.
```

Comment:
This still sounds awkward. Perhaps: "You can get the email configuration value by calling .... "